src/main/java/org/folio/service/consortia/ConsortiaServiceImpl.java

```diff
@@ -0,0 +1,95 @@
+package org.folio.service.consortia;
+
+import io.vertx.core.Future;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpStatus;
+import org.folio.okapi.common.XOkapiHeaders;
+import org.folio.rest.exception.UnprocessableEntityException;
+import org.folio.rest.exception.UnprocessableEntityMessage;
+import org.folio.rest.jaxrs.model.User;
+import org.folio.rest.tools.client.Response;
+import org.folio.rest.util.OkapiConnectionParams;
+import org.folio.rest.util.RestUtil;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
+import static org.folio.rest.impl.BLUsersAPI.FORGOTTEN_PASSWORD_FOUND_INACTIVE;
+import static org.folio.rest.impl.BLUsersAPI.LOCATE_USER_USERNAME;
+import static org.folio.rest.impl.BLUsersAPI.LOCATE_USER_PHONE_NUMBER;
+import static org.folio.rest.impl.BLUsersAPI.LOCATE_USER_EMAIL;
+import static org.folio.rest.impl.BLUsersAPI.LOCATE_USER_MOBILE_PHONE_NUMBER;
+
+public class ConsortiaServiceImpl implements ConsortiaService {
```

Comment:
As I know there was some conversation to not use consortia in naming, because mod-consortia is disabled by default.
I suggest to rename to CrossTenantUserServiceImpl.
@okolawole-ebsco is it actual?

Reply:
Yes, i think we should use "cross tenant" instead of consortia.